examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -250,190 +345,62 @@ def unpackage_dict(inputs):
 the model are expected to be in the range `[0, 255]`.
 """
 
-model = keras_cv.models.RetinaNet(
-    # number of classes to be used in box classification
-    classes=len(class_ids),
-    # For more info on supported bounding box formats, visit
-    # https://keras.io/api/keras_cv/bounding_box/
-    bounding_box_format="xywh",
-    # KerasCV offers a set of pre-configured backbones
-    backbone="resnet50",
-    # Each backbone comes with multiple pre-trained weights
-    # These weights match the weights available in the `keras_cv.model` class.
-    backbone_weights="imagenet",
-    # include_rescaling tells the model whether your input images are in the default
-    # pixel range (0, 255) or if you have already rescaled your inputs to the range
-    # (0, 1).  In our case, we feed our model images with inputs in the range (0, 255).
-    include_rescaling=True,
-)
-# Fine-tuning a RetinaNet is as simple as setting backbone.trainable = False
-model.backbone.trainable = False
-
-metrics = [
-    keras_cv.metrics.COCOMeanAveragePrecision(
-        class_ids=range(21),
-        bounding_box_format="xywh",
-        name="Mean Average Precision",
-    ),
-    keras_cv.metrics.COCORecall(
-        class_ids=range(21),
+with strategy.scope():
+    model = keras_cv.models.RetinaNet(
+        # number of classes to be used in box classification
+        classes=len(class_ids),
+        # For more info on supported bounding box formats, visit
+        # https://keras.io/api/keras_cv/bounding_box/
         bounding_box_format="xywh",
-        max_detections=100,
-        name="Recall",
-    ),
-]
-optimizer = tf.optimizers.SGD(global_clipnorm=10.0)
+        # KerasCV offers a set of pre-configured backbones
+        backbone="resnet50",
+        # Each backbone comes with multiple pre-trained weights
+        # These weights match the weights available in the `keras_cv.model` class.
+        backbone_weights="imagenet",
+        # include_rescaling tells the model whether your input images are in the default
+        # pixel range (0, 255) or if you have already rescaled your inputs to the range
+        # (0, 1).  In our case, we feed our model images with inputs in the range (0, 255).
+        include_rescaling=True,
+    )
+    # Fine-tuning a RetinaNet is as simple as setting backbone.trainable = False
+    model.backbone.trainable = False
+
+    metrics = [
+        keras_cv.metrics.COCOMeanAveragePrecision(
+            class_ids=range(21),
```

Comment:
maybe we should change 21 to len(class_ids)

Reply:
Actually no longer needed. Removed.